scripts/convertMarlinSteeringToGaudi.py

```diff
@@ -88,6 +94,10 @@ def getExecutingProcessors(lines, tree):
   execGroup = tree.findall('group')
 
   for proc in execProc:
+    if proc.tag == "if":
+      for child in proc:
+        if child.tag == "processor":
+          lines.append("# algList.append(%s)" % child.get('name').replace(".", "_"))
```

Comment:
would it be possible to do 
```suggestion
          lines.append("# algList.append(%s)  # %s" % (child.get('name').replace(".", "_"), <VALUE OF THE IF TAG>))
```
?

Reply:
yes, I'll add it.